### PR TITLE
Release/v5.0.0

### PR DESCRIPTION
**Pull request by HarrisonHough**
## Description

-   Briefly describe what this change will do

## How to Test

-   Add steps to locally test these changes

## Checklist

-   [ ] Tests written or updated for the changes.
-   [ ] Documentation is updated.
-   [ ] Changelog is updated.

### CODE REVIEW

File: Samples~/AvatarCreatorSamples/LegacyAvatarCreator/Scripts/ProfileManager.cs [hunk 1]
```diff
@@ -30,10 +31,9 @@ private void OnEnable()
             AuthManager.OnSignedOut += DeleteSession;
         }
 
-
         private void OnDisable()
         {
-            profileUI.SignedOut -= AuthManager.Logout;
+            profileUI.SignedOut += AuthManager.Logout;
```
Comment (rk132) [inline Samples~/AvatarCreatorSamples/LegacyAvatarCreator/Scripts/ProfileManager.cs:36]: Hmm, shouldn't we remove it, not add it here?

File: Runtime/AvatarCreator/Scripts/Managers/AvatarManager.cs [hunk 2]
```diff
@@ -13,27 +14,20 @@ namespace ReadyPlayerMe.AvatarCreator
...
-        /// <param name="gender">Gender of avatar</param>
         /// <param name="avatarConfig">Config for downloading preview avatar</param>
         /// <param name="token">Cancellation token</param>
-        public AvatarManager(BodyType bodyType, OutfitGender gender, AvatarConfig avatarConfig = null, CancellationToken token = default)
+        public AvatarManager(AvatarConfig avatarConfig = null, CancellationToken token = default)
         {
-            this.bodyType = bodyType;
-            this.gender = gender;
-
             if (avatarConfig != null)
```
Comment (rk132) [inline Runtime/AvatarCreator/Scripts/Managers/AvatarManager.cs:31]:
```suggestion
        /// <param name="gender">Gender of the avatar, that is being modified</param>
        public AvatarManager(AvatarConfig avatarConfig = null, CancellationToken token = default, OutfitGender gender = OutfitGender.None)
        {
            this.gender = gender;
            if (avatarConfig != null)
```

File: Samples~/AvatarCreatorSamples/LegacyAvatarCreator/Scripts/UI/SelectionScreens/AvatarCreatorSelection.cs [hunk 3]
```diff
@@ -64,8 +64,6 @@ private async void Setup()
             ctxSource = new CancellationTokenSource();
 
             avatarManager = new AvatarManager(
-                AvatarCreatorData.AvatarProperties.BodyType,
-                AvatarCreatorData.AvatarProperties.Gender,
                 inCreatorConfig,
                 ctxSource.Token);
```
Comment (rk132) [inline Samples~/AvatarCreatorSamples/LegacyAvatarCreator/Scripts/UI/SelectionScreens/AvatarCreatorSelection.cs:68]:
```suggestion
            avatarManager = new AvatarManager(
                inCreatorConfig,
                ctxSource.Token, AvatarCreatorData.AvatarProperties.Gender);
```

**Review by rk132 — CHANGES_REQUESTED** [review]

File: Samples~/AvatarCreatorSamples/LegacyAvatarCreator/Scripts/ProfileManager.cs (hunk 1 above)
Comment (HarrisonHough) [reply Samples~/AvatarCreatorSamples/LegacyAvatarCreator/Scripts/ProfileManager.cs:36]: yep good catch

**Pull request merged**